docs/pages/material-ui/api/accordion.json

```diff
@@ -90,12 +96,6 @@
       "description": "Styles applied to the root element unless `disableGutters={true}`.",
       "isGlobal": false
     },
-    {
-      "key": "region",
-      "className": "MuiAccordion-region",
-      "description": "Styles applied to the region element, the container of the children.",
-      "isGlobal": false
-    },
```

Comment:
@sai6855 Why was this `region` CSS class removed from the API docs? It's no longer there: https://mui.com/material-ui/api/accordion/#classes

Reply:
But isn't that incorrect? Shouldn't it be in both? We have it in both [Accordion classes type](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Accordion/accordionClasses.ts#L17-L18) and in [Accordion slots type](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Accordion/Accordion.d.ts#L27-L31).